homeassistant/components/tesla_wall_connector/sensor.py

```diff
@@ -125,6 +125,13 @@ class WallConnectorSensorDescription(
         state_class=SensorStateClass.MEASUREMENT,
         entity_category=EntityCategory.DIAGNOSTIC,
     ),
+    WallConnectorSensorDescription(
+        key="session_energy_wh",
+        translation_key="session_energy_wh",
+        native_unit_of_measurement=UnitOfEnergy.WATT_HOUR,
+        value_fn=lambda data: data[WALLCONNECTOR_DATA_VITALS].session_energy_wh,
+        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
Only thing I have a doubt about is the chosen state class as this looks like a value that's only increasing and reset when a new session occurs.

Reply:
IMO, it must be `measurement` because the absolute value in the interesting value. `total` and `total_increasing` are only used when accumulated growth or decline is interesting.

Reply:
Okay, I will make that change.

Reply:
Sorry I misunderstood earlier. Resolving this conversation as no action required.